README.md

```diff
@@ -68,10 +68,12 @@ The library depends on code that has been generated from Symphony's YAML spec fi
 
 In case there is a need to regenerate the code from Symphony's YAML specs:
 
- 1. Make sure Node Package Manager is installed
- 2. Install nswag via command line:  `npm install nswag -g`
- 3. Install api-spec-converter via command line: `npm install api-spec-converter -g`
- 4. Run SymphonyOSS.RestApiClient\Generated\OpenApi\generate.bat
+ 1. Clone [this](https://github.com/mvu02/NSwag) repo to get a local copy of the NSwag code
```

Comment:
We should revert these changes if the NSwag PR is accepted before we merge this one

Reply:
We could scrap this whole PR if NSwag PR is accepted before we merge this one and jstu do a clean rebuild using nswag npm.